eoscostarica/guide.eoscostarica.io
# docs(blog-posts.md): create new doc

### Blog Posts Page

#81 

### Steps to test
1. Run project locally
1. Go to community resources and see Blog Posts
1.

#### CheckList
